shared/gh/partials/agenda-view.html

```diff
@@ -0,0 +1,108 @@
+<%
+    prevDate = null;
+
+    /**
+     * Compare two dates to each other and return true if the year, month and day match each other
+     *
+     * @param  {Date}    _prevDate    The first Date to match against
+     * @param  {Date}    currDate     The second Date to match against
+     * @return {Boolean}              Return `true` if the dates match
+     * @private
+     */
+    var isADifferentDay = function(_prevDate, currDate) {
+        if (moment(_prevDate).format('YYYYMMDD') === moment(currDate).format('YYYYMMDD')) {
```

Comment:
This can probably be written as a oneliner: `return (moment(_prevDate).format('YYYYMMDD') === moment(currDate).format('YYYYMMDD'));`